picard/ui/options/releases.py

```diff
@@ -74,7 +74,10 @@ def showEvent(self, event):
     def hideEvent(self, event):
         super().hideEvent(event)
         if not IS_WIN:
-            self.valueChanged.disconnect(self.show_tip)
+            try:
+                self.valueChanged.disconnect(self.show_tip)
+            except TypeError:
+                pass
```

Comment:
You may want to use https://github.com/metabrainz/picard/blob/master/picard/util/__init__.py#L560
It is meant to handle those cases (plus reconnects).

Reply:
Actually this does not really fit here. We want only the disconnect here, not reconnecting. This is just to ensure `self.show_tip` does not get triggered while the view is hidden.